src/main/java/edu/ucsb/cs156/organic/entities/Student.java

```diff
@@ -0,0 +1,29 @@
+package edu.ucsb.cs156.organic.entities;
+
+import lombok.*;
+
+import javax.persistence.*;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Builder
+@Entity(name = "entstudent")
```

Comment:
camelCase?

Reply:
That's actually a typo... should just be student.  Good catch.. I'll fix it.

Reply:
Fixed